smithy-diff/src/main/java/software/amazon/smithy/diff/evaluators/RemovedShape.java

```diff
@@ -19,19 +19,23 @@
 import java.util.stream.Collectors;
 import software.amazon.smithy.diff.Differences;
 import software.amazon.smithy.model.shapes.Shape;
+import software.amazon.smithy.model.traits.EnumTrait;
 import software.amazon.smithy.model.traits.PrivateTrait;
 import software.amazon.smithy.model.validation.ValidationEvent;
 
 /**
- * Creates an ERROR event when a non-private shape is removed.
+ * Creates an ERROR event when a non-private non-scalar shape is removed.
+ * Creates a NOTE event when a non-private scalar shape is removed.
  */
 public final class RemovedShape extends AbstractDiffEvaluator {
     @Override
     public List<ValidationEvent> evaluate(Differences differences) {
         return differences.removedShapes()
                 .filter(shape -> !shape.hasTrait(PrivateTrait.class))
                 .filter(shape -> !isMemberOfRemovedShape(shape, differences))
-                .map(shape -> error(shape, String.format("Removed %s `%s`", shape.getType(), shape.getId())))
+                .map(shape -> isScalarType(shape)
+                        ? note(shape, String.format("Removed %s `%s`", shape.getType(), shape.getId()))
```

Comment:
Wonder if this should be a `warning` instead.

Reply:
This really should be a warning. We need to bring this to people's attention but not fail the build, hence a warning.